### DISCUSSION TITLE
Registration and Segmentation

### DISCUSSION THREAD

**Discussion by Alhonainy21** [FeTS-AI/Front-End#71] [Q&A]
Hello,
I have two questions please:
1: In some tools used for Medical Imaging, there are two distinct steps: registration and segmentation. Are these two processes combined in the following command for FeTS?

${fets_root_dir}/bin/FeTS_CLI_Segment -d ... etc

2: Is the above mentioned command pass the same parameteres: ( -d ) the directory containing the data, (-a) pre-trained models, (-lF,-g and -t) , when i use docker image?

Thank you!

**Comment by sarthakpati** [comment]
1. Processing the data is a separate step, which produces a directory `DataForFeTS`: https://fets-ai.github.io/FL-Pre/process_data
2. The `DataForFeTS` location is the input for subsequent steps: https://fets-ai.github.io/FL-Pre/runningApplication

Hope this helps!
